runtime/description.js

```diff
@@ -30,7 +30,7 @@ export default class Description {
       return desc;
     }
 
-    return this._recipe.name;
+    return this._recipe.name || 'Anonymous recipe with no UI';
```

Comment:
sorry for nitpicking, but "with no UI" is not accurate, the recipe may have particles that produce UI, but don't have any description configured :)